test/plain-list/AutosuggestApp.test.js

```diff
@@ -580,7 +580,11 @@ describe('Default Autosuggest', () => {
 
     it('should be called with the right parameters', () => {
       focusAndSetInputValue('e');
-      expect(shouldRenderSuggestions).to.be.calledWithExactly('e');
+      expect(shouldRenderSuggestions.callCount).to.equal(4);
```

Comment:
For completeness, you might wanna test onBlur and the others too.

Reply:
Good idea, done :+1:

Reply:
Cool!